RecoTracker/LSTCore/src/alpaka/LSTEvent.h

```diff
@@ -95,10 +93,10 @@ namespace ALPAKA_ACCELERATOR_NAMESPACE::lst {
           nEndCapMap_(deviceESData->nEndCapMap),
           modules_(*deviceESData->modules),
           pixelMapping_(*deviceESData->pixelMapping),
-          endcapGeometry_(*deviceESData->endcapGeometry) {
-      initSync(verbose);
-    }
-    void resetEventSync();  // synchronizes
+          endcapGeometry_(*deviceESData->endcapGeometry),
+          addObjects_(verbose) {}
+    void initSync(bool verbose);  // synchronizes, for standalone usage
```

Comment:
I recall there was a comment for why do we need to sync here. I'm not sure I had a good answer. Perhaps it's OK to drop the sync then

Reply:
Standalone doesn't work in that case, I think. Because in the standalone we create `LSTEvent` objects in the beginning of the loop and then reuse them. I didn't want to redesign the structure of the standalone scripts, so I thought that what I did (removing the syncs from CMSSW while keeping the functions) was a good middle ground solution.

Reply:
uhm, it sounds like a `bool sync` may be needed to address that comment

Reply:
Within CMSSW, `initSync` is not needed at all, so I don't see the point of adding the `bool`.
...And looking closer, I also need to remove this `bool` (it is now useless).